File: web/src/entities/happening.js
export default class Happening {
  constructor ({ id, name, accountId, description, organizerDescription, agenda, isPublished }) {
    this.id = id
    this.name = name
    this.accountId = accountId
    this.description = description
    this.organizerDescription = organizerDescription
    this.agenda = agenda
    this.isPublished = isPublished
  }
}

File: ios/Happ/Models/Components/Happening/Happening.swift
//
//  Happening.swift
//  Happ
//
//  Created by Maksymilian Galas on 17/09/2020.
//  Copyright © 2020 Infinity Pi Ltd. All rights reserved.
//

import Foundation

struct Happening: ResponseData, Identifiable, Hashable {
    let id: String
    let name: String
    let accountId: String
    let description: String
    let organizerDescription: String
    let agenda: String
    let isPublished: Bool
}
